alembic/config.py

```diff
@@ -495,6 +495,8 @@ def _get_toml_config_value(
                     "dict[str, str]",
                     {k: v % (self.toml_args) for k, v in value.items()},
                 )
+            elif isinstance(value, int):
+                value = str(value)
```

Comment:
why str()?   truncate_slug_length should be able to receive int directly from config (it looks like it calls int() on it, expecting to receive it from alembic.ini, but that's fine).

since toml is giving us an int, we should move forward not unnecessarily converting to string and back.